Makefile

```diff
@@ -168,7 +168,7 @@ fixtures/%:
 	if [ ! -d "opentelemetry-helm-charts" ]; then \
 		git clone https://github.com/open-telemetry/opentelemetry-helm-charts.git; \
 	fi
-	helm install test -f .github/workflows/e2e/k8s/collector-helm-values.yml opentelemetry-helm-charts/charts/opentelemetry-collector
+	helm install test -f .github/workflows/e2e/k8s/collector-helm-values.yml opentelemetry-helm-charts/opentelemetry-collector
```

Comment:
@RonFed I missed the change in https://github.com/open-telemetry/opentelemetry-go-instrumentation/commit/eb597d65663a332f208bb8f31ffb8838e2d89547#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L160 that updated the `helm install` command to add `/charts/`, but it broke for me with 

```
Error: INSTALLATION FAILED: failed to download "opentelemetry-helm-charts/charts/opentelemetry-collector"
```

I couldn't figure out how to fix it, I'm assuming it does work since the e2e tests still work, but do you know why you added it or what we need to do locally to get the new command to work?